Lib/test/test_iter.py

```diff
@@ -302,7 +302,7 @@ def __eq__(self, other):
             # listiter_reduce_general
             self.assertEqual(
                 run("reversed", orig["reversed"](list(range(8)))),
-                (iter, ([],))
+                (reversed, ([],))
```

Comment:
This test was added from https://github.com/python/cpython/pull/101769, not sure that it will be okay to change the implementation detail.

Reply:
I think it is OK, see my other comment.